01_fib/baijie/fib_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := strconv.Quote("1\n")
+	actual := strconv.Quote(buf.String())
```

Comment:
Let me know if I'm wrong, but I don't think using `strconv.Quote` here and above adds value.

Reply:
Done